app/controllers/families_controller.rb

```diff
@@ -18,8 +18,17 @@ def create
   end
 
   def destroy
-    @family = Family.find(params[:id]).delete
-    
+    family.delete
+
+    respond_with @family, location: -> { families_path }
+  end
```

Comment:
Usually `destroy` is the last action in the controller. What do you think about moving it to be last?

Reply:
Done with fixup 01ae900